src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -837,14 +912,24 @@ public interface IConversionExpression : IHasOperatorMethodExpression
         /// Value to be converted.
         /// </summary>
         IOperation Operand { get; }
+
         /// <summary>
         /// Kind of conversion.
         /// </summary>
         ConversionKind ConversionKind { get; }
+
         /// <summary>
-        /// True if and only if the conversion is indicated explicity by a cast operation in the source code.
+        /// Gets the <see cref="IConversion"/> instance that provides more information about what
+        /// sort of conversion this was.
         /// </summary>
-        bool IsExplicit { get; }
+        IConversion Conversion { get; }
```

Comment:
Since this also returns a method symbol, it creates a duplication of information since IConversionExpression also implements IHasOperatorMethodExpression, which is supposed to expose the same information about the method. Perhaps IConversionExpression should no longer inherit from IHasOperatorMethodExpression.